pkg/network/common/egress_dns.go

```diff
@@ -98,17 +102,19 @@ func (e *EgressDNS) Delete(policy networkv1.EgressNetworkPolicy) {
 	}
 }
 
-func (e *EgressDNS) Update(dns string) (bool, error) {
-	e.lock.Lock()
-	defer e.lock.Unlock()
+func (e *EgressDNS) Update(dns string) {
```

Comment:
if this is only used internally (which it seems like it is?) it should have a lowercase name